hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -268,6 +268,13 @@
       datanode periodically send node report to SCM. Unit could be
       defined with postfix (ns,ms,s,m,h,d)</description>
   </property>
+  <property>
+    <name>hdds.statemachine.endpoint.task.thread.count</name>
+    <value>2</value>
+    <tag>OZONE, DATANODE, MANAGEMENT</tag>
+    <description>Maximum number of threads in the thread pool that Datanode
+      will use for GETVERSION, REGISTER, HEARTBEAT to SCMs.</description>
+  </property>
```

Comment:
Can we add more details about what's the proper value for this proerpty if user want to change it?

Reply:
I remove this config, and get thread pool size by add number of scm and recon.